tests/test_webhooks.py

```diff
@@ -1,19 +1,17 @@
-import pytest
+from up42 import webhooks
 
-from up42.webhooks import Webhook
-
-from .fixtures.fixtures_globals import WEBHOOK_ID, WORKSPACE_ID
+from .fixtures import fixtures_globals as constants
 
 
 def test_webhook_initiate(webhook_mock):
-    assert isinstance(webhook_mock, Webhook)
-    assert webhook_mock.webhook_id == WEBHOOK_ID
-    assert webhook_mock.workspace_id == WORKSPACE_ID
+    assert isinstance(webhook_mock, webhooks.Webhook)
+    assert webhook_mock.webhook_id == constants.WEBHOOK_ID
+    assert webhook_mock.workspace_id == constants.WORKSPACE_ID
 
 
 def test_webhook_info(webhook_mock):
-    assert webhook_mock.info
-    assert webhook_mock._info["id"] == WEBHOOK_ID
+    info = webhook_mock.info
+    assert info and info["id"] == constants.WEBHOOK_ID
```

Comment:
will `info and info["id"]` not always return `info["id"]`?

If that's the case, consider updating assertion to
`assert info["id"] == constants.WEBHOOK_ID`

Reply:
info and info["id"] always return info["id"] since it always uses the last value if the first one can be evaluated as True